web/vtadmin/src/components/NavRail.module.scss

```diff
@@ -27,9 +27,11 @@ $navRailHoverTransition: background-color 0.1s ease-in-out;
 }
 
 .logoContainer {
-    display: block;
+    display: flex;
```

Comment:
These changes are also reflected in a PR to OS VTAdmin: https://github.com/vitessio/vitess/pull/9866